samples/net/rpl_border_router/prj_frdm_k64f_mcr20a_tc.conf

```diff
@@ -0,0 +1,153 @@
+# Generic IP stack options and features
```

Comment:
"tc" in prj_frdm_k64f_mcr20a_tc.conf filename is not clear at all. Another file is  prj_frdm_k64f_mcr20a_mbedtls.conf, why can't we spell "tinycrypt" here too?